lib/spanner/command/request.ex

```diff
@@ -48,10 +56,25 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":", parts: 2)
-    cmd_config_file = Path.join([config_path, bundle, Config.dynamic_file_name()])
-    case File.exists?(cmd_config_file) do
+    cmd_config_dir = Path.join([config_path, bundle])
+
+    case File.dir?(cmd_config_dir) do
       true ->
-        Config.Parser.read_from_file(cmd_config_file)
+        case Config.find_config(cmd_config_dir) do
+          [] ->
+            # If a directory for dynamic config exists but there are no config files
+            # we should warn the user
+            {:not_found, cmd_config_dir}
+          [config_path | []] ->
+            with {:ok, config} <- Config.parser.read_from_file(config_path) do
+              {:ok, config}
+            end
+          [config_path | _] ->
+            # If multiple configs are found we should warn the user
+            with {:ok, config} <- Config.Parser.read_from_file(config_path) do
+              {:multiple_configs, config, config_path}
+            end
+        end
```

Comment:
Yeah I think this is ok. Although it might be nice to also return `:not_found` even when the directory doesn't exist (in the clause below this)